### PR TITLE
Improve kafka distinct_id messages

### PR DESCRIPTION
**Pull request by yakkomajuri**
## Changes

Only call `queueMessage` once, stop passing Postgres row id to CH table

## Checklist

-   [ ] Updated Settings section in README.md, if settings are affected
-   [ ] Jest tests

### CODE REVIEW

**Comment by yakkomajuri** [comment]
Need to update some other things here

File: src/utils/db/db.ts [hunk 1]
```diff
@@ -560,7 +560,7 @@ export class DB {
                         WHERE person_id='${escapeClickHouseString(person.uuid)}'
                           AND team_id='${person.team_id}'
                           AND is_deleted=0
-                        ORDER BY id`
+                        ORDER BY _offset`
```
Comment (yakkomajuri) [inline src/utils/db/db.ts:563]: this only affects tests

**Comment by yakkomajuri** [comment]
This also fixes tests that are currently broken

**Comment by yakkomajuri** [comment]
Stamp of approval @Twixes ?

**Review by Twixes — APPROVED** [review]
Stamp of approval

**Comment by yakkomajuri** [comment]
Thanks for proactively lending a hand here ❤️

**Pull request merged**